cmd/controller/app/options/options.go

```diff
@@ -358,11 +359,8 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 }
 
 func (o *ControllerOptions) Validate() error {
-	switch o.DefaultIssuerKind {
-	case "Issuer":
-	case "ClusterIssuer":
-	default:
-		return fmt.Errorf("invalid default issuer kind: %v", o.DefaultIssuerKind)
+	if len(o.DefaultIssuerKind) == 0 {
+		return errors.New("a default issuer kind must be set")
```

Comment:
Sorry to be a pain, but maybe we should also put the flag name in the error string?